.circleci/config.yml

```diff
@@ -160,6 +160,8 @@ workflows:
           filters:
             tags:
               only: /.*/
+            branches:
+              only: master
```

Comment:
i made the docker build workflow in ci no longer required since it does nothing on PRs anyway. and this change also makes it so it doesnt run at all expect when we merge master and tag